Dockerfile

```diff
@@ -39,7 +39,13 @@ LABEL name="MongoDB Atlas Operator" \
       vendor="MongoDB" \
       release="1" \
       summary="MongoDB Atlas Operator Image" \
-      description="MongoDB Atlas Operator is a Kubernetes Operator allowing to manage MongoDB Atlas resources not leaving Kubernetes cluster"
+      description="MongoDB Atlas Operator is a Kubernetes Operator allowing to manage MongoDB Atlas resources not leaving Kubernetes cluster" \
+      io.k8s.display-name="MongoDB Atlas Operator" \
+      io.k8s.description="MongoDB Atlas Operator is a Kubernetes Operator allowing to manage MongoDB Atlas resources not leaving Kubernetes cluster" \
+      io.openshift.tags="mongodb,atlas" \
+      io.openshift.maintainer.product="MongoDB" \
```

Comment:
just asking: if this image is not only for OpenShift, is it ok to always have it? (*.openshift.* fields)

Reply:
I guess that's okay. This info is getting extracted and displayed only in Openshift.